redis_broadcast.go

```diff
@@ -172,8 +180,10 @@ func (bc *redisBroadcast) AllRooms() []string {
 	req.done = make(chan bool, 1)
 
 	bc.requests[req.RequestID] = &req
-	bc.pub.Conn.Do("PUBLISH", bc.reqChannel, reqJSON)
-
+	_, err := bc.pub.Conn.Do("PUBLISH", bc.reqChannel, reqJSON)
+	if err != nil {
+		return []string{} // if error occurred,return empty
+	}
```

Comment:
after that need break row

Reply:
sorry,unnecessary line breaks is not allowed in my work,i changed it